docs/geps/gep-01.rst

```diff
@@ -112,10 +124,11 @@ Parameters of the taxes and transfers system
 The structure of these parameters will be laid out in gep-3; we just note some
 general naming considerations here.
 
-* There is a hierarchical structure to these parameters in that each of them is
+- There is a hierarchical structure to these parameters in that each of them is
   associated with a group (e.g., ``arbeitsl_geld``, ``kinderzuschlag``). These groups or
   abbreviations thereof do not re-appear in the name of the parameter.
-
+- Parameter names should be aligned with relevant column names. This makes searching for
+  identifiers easier and less error-prone.
 
 
 Python Identifiers (Functions, Variables)
```

Comment:
A bit contradictory that we mention functions here (which typically are also columns/potential targets and should be written in German.

Maybe mention helper functions instead?

Reply:
Great. Using 'other functions' now.